src/components/bookmarks/BookmarkHeader.tsx

```diff
@@ -30,17 +30,9 @@ export const BookmarkHeader = ({ id, suffix, query, style }: Props) => {
       accessibilityHint="Zu der Lesezeichenliste hinzufügen"
     >
       <Icon
-        size={normalize(22)}
+        size={normalize(44)}
```

Comment:
unfortunately this does not change anything, as `size` is ignored when there is xml present. (i missed that as well, when i first added this icon)

furthermore you are setting it to the default value (which you set in another Icon.js).